facebook4j-core/src/main/java/facebook4j/internal/json/PageJSONImpl.java

```diff
@@ -86,7 +87,24 @@ private void init(JSONObject json) throws FacebookException {
             link = getURL("link", json);
             isPublished = getBoolean("is_published", json);
             canPost = getBoolean("can_post", json);
-            likes = getInt("likes", json);
+
+            if (!json.isNull("likes")) {
```

Comment:
This code may be extracted since used multiple times already.
If you would suggest how to do it I may fix it in this PR.